modules/base-commerce/composables/useLocation.ts

```diff
@@ -0,0 +1,15 @@
+import { onMounted, ref } from 'vue'
+
+export default () => {
+  const location = ref(null)
```

Comment:
Could you please add TS definition so that there is an autocomplete when using the useLocation composable?